css/properties/text-align-last.json

```
@@ -36,8 +36,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": false,
-              "notes": "See WebKit <a href='https://webkit.org/b/76173'>bug 76173</a>."
+              "version_added": "16.0"
```

This should fix the lint failure:

```suggestion
              "version_added": "16"
```